sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/SaveAsHiveFile.scala

```diff
@@ -125,10 +162,23 @@ private[hive] trait SaveAsHiveFile extends DataWritingCommand {
     val stagingDir = hadoopConf.get("hive.exec.stagingdir", ".hive-staging")
     val scratchDir = hadoopConf.get("hive.exec.scratchdir", "/tmp/hive")
 
+    // Hive sets session_path as HDFS_SESSION_PATH_KEY(_hive.hdfs.session.path) in hive config
+    val sessionScratchDir = externalCatalog.unwrapped.asInstanceOf[HiveExternalCatalog]
+      .client.getConf("_hive.hdfs.session.path", "")
```

Comment:
Could you add a new conf under the name spark `spark.sql.hive.` instead of the same name with hive? Also, please follow the naming rule: https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala#L20-L47

By doing so, [sql/create-docs.sh](https://github.com/apache/spark/blob/master/sql/create-docs.sh) could automatically generate a doc for the conf.